ibis/backends/base/__init__.py

```diff
@@ -155,8 +155,7 @@ def list_tables(self, like: str = None, database: str = None) -> List[str]:
 
     # @abc.abstractmethod
     def table(self, name: str, database: str = None) -> ir.TableExpr:
-        """
-        """
+        """ """
```

Comment:
should prob just kill these (can be another PR)

Reply:
Yup.